custom/icds_reports/static/icds_reports/mobile/css/map-chart.less

```diff
@@ -17,7 +17,7 @@
   .float-unset;
   .p-3;
   .rounded-border;
-  height: calc(100vh - 170px);
+  height: ~"calc(100vh - 170px)";
```

Comment:
Does the `calc` on line 46 also need to be updated?

Reply:
thanks, not sure how I missed that.

https://github.com/dimagi/commcare-hq/pull/26212/commits/67c4abed33b90bdab6599af8c38794ea24b3ce09